util/tests/quantity/test_storage.py

```diff
@@ -189,37 +136,4 @@ def test_numpy_data_becomes_cupy_with_gpu_backend(
         units=units,
         gt4py_backend=gt4py_backend,
     )
-    assert isinstance(quantity.data, cupy.ndarray)
-    assert isinstance(quantity.storage, gt4py.storage.storage.GPUStorage)
-
-
-@pytest.mark.parametrize("backend", ["gt4py_numpy"], indirect=True)
```

Comment:
I guess if you pass a cupy array on a cpu backend it just performs a data copy and vice-versa? If so that should be fine.